components/Card/card.css

```diff
@@ -1,6 +1,6 @@
 .card {
   width: 20rem;
-
+  border: 1px solid gray;
```

Comment:
suggestion for not hard-coding the color. Making it a variable makes it more reusable and uniform throughout the site.

Reply:
I have done the changes thanks for the suggeston will be merging the branch now.